src/tooltip/index.tsx

```diff
@@ -23,12 +37,14 @@ export interface TooltipProps {
   leaveDelay?: number;
   /** Manually control the open state */
   open?: boolean;
-  /** Whether or not to show an arrow on the Tooltip. Defaults to `false`. */
-  showArrow?: boolean;
```

Comment:
Is this no longer possible to support or?

Reply:
I can't really find any documentation for it.. but MCW do expose some methods for controlling a "caret". However, I think it is only for rich tooltips and there is no documentation (that I can find)